### DISCUSSION TITLE
GitHub Login

### DISCUSSION THREAD

**Discussion by kumar-z** [community/community#35634] [Other Feature Feedback, Questions, & Ideas]
I'm unable to login into my GitHub via the 2FA code. I don't have the  recovery keys also. When tried resetting the account the option to recover my account seems to be disabled -- 
![image](https://user-images.githubusercontent.com/55299856/194798692-b6bd8590-9550-44d8-ba54-6380cbf3a679.png)

Please help.

**Comment by MitkoVtori** [comment]
You can try finding your problem [here](https://support.github.com/?q=2fa)

**Reply by MitkoVtori** [reply]
or if it doesn't work, try [here](https://docs.github.com/en/authentication/managing-commit-signature-verification)
